docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -191,10 +191,32 @@ ClickHouse        Trino             Notes
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _clickhouse-sql-support:
+
+SQL support
```

Comment:
Is it possible to extract cut-and-paste movements to separate commit? It makes it easier to notice things were just moved and content was not changed.

Reply:
I can try if you think it's necessary, it might just be a mess since I'm a git amateur 😅

Reply:
Not required, we can try that for future changes. 😄